src/decideid_assets/app/components/header/Header.tsx

```diff
@@ -1,15 +1,13 @@
-import React, { useContext } from 'react';
+import React, { useContext } from "react";
```

Comment:
you have a different setup locally that will change the format of the tsx file ?